public/api/conf/1.0/schemas/dereferenced/fullReturnSchema.json

```diff
@@ -690,7 +690,7 @@
             "additionalProperties": false
         },
         "submissionType": {
-            "description": "Indicate whether this is the first (original) return or an amended (revised) return",
+            "description": "Indicate if this is a revised return [this should be a boolean true or false]. Limited to the following possible values",
```

Comment:
This isn't a boolean so I think we need to remove that bit in the square bracers. I also think the last sentence might be duplicated on the webpage after parsing